docs/l4-l7.rst

```diff
@@ -230,7 +240,9 @@ The new application is created on this BIG-IP device in the specified partition,
 .. note::
 
    - On clicking :guilabel:`Submit`, the application is created asynchronously in the background. Once the operation is successful, the app UI loads the latest data. For more details, refer to https://clouddocs.f5.com/f5-aci-servicecenter/latest/l4-l7.html#as3-async-task-processing
-   - For :guilabel:`service-discovery/http` template, You can select APIC Tenant, Application, Endpoint Group from the provided drop-downs to associate them with the BIG-IP Application’s pool. Whenever the APIC Endpoint Group will have a new endpoint attached or detached from it, the FAST application’s pool will also have the same set of endpoints in it on the BIG-IP.
+   - For :guilabel:`service-discovery/http` template, You can select APIC Tenant, Application, Endpoint Group from the provided drop-downs to associate them with the BIG-IP Application’s pool. The VRF field is a read-only field and is retrieved from the APIC’s Endpoint Group itself. You can select Route Domain from the dropdown options to associate them with the Application's pool. Whenever the APIC Endpoint Group will have a new endpoint attached or detached from it, the FAST application’s pool will also have the same set of endpoints in it on the BIG-IP.
```

Comment:
For the :guilabel:`service-discovery/http' template, you can select the APIC Tenant, Application, and Endpoint Group from the drop-down lists to associate...